doc/integrator/querier.rst

```diff
@@ -25,16 +25,16 @@ The web service configuration is done in the ``vars_<project>.yaml`` file:
                         separator: ","
 
 The ``dbsession: "<session name>"`` in the ``enum`` is just a shortcut
-if almost of the session are similar in the layers. If the dbsession isn't
-defined we use the main DB session.
+if almost of the session are similar in the layers. If ``dbsession`` is not
+defined, the main DB session is used.
 
 The ``table: "<[schema.]table name>"`` in the layer is just a shortcut
-if almost of the attribute of the layer are in the same table.
-Each attributes requires a table.
+if almost of the attributes of the layer are in the same table.
+Each attribute requires a table.
 
-If the ``column_name`` isn't defined we use the attribute name.
+If the ``column_name`` is not defined, the attribute name is used.
 
-If the ``separator`` is defined we consider the column as a list of values.
+If the ``separator`` is defined, the column is considered as a list of values.
```

Comment:
What is "the column"? Do you mean ":" (colon)? https://en.wikipedia.org/wiki/Colon_(punctuation)